product-definitions/input-to-draft-regulation.md

```diff
@@ -157,8 +157,10 @@
           Embedded browsers are browsers that are intended for integration into another system or application.
         </p>
       </td>
-      <td><!-- Comments --></td>
-      <td><!-- Proposed change --></td>
+      <td>Command-line tools (such as Curl) and other web clients used for software development and testing also "enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet" through the <a href="https://en.wikipedia.org/wiki/Command-line_interface">command-line interface</a> or through other means. Yet they are not browsers per se. In particular they do not <em>render</em> the web content, but merely fetch it. They are used by technically savvy users to perform specific tasks (like looking at HTTP headers for example) and their interaction capabilities are limited. We suggest amending the description to account for this.</td>
+      <td><p>In the context of this category of products, browsers are software products with digital elements that fetch and render web content hosted on servers that are connected to networks such as the Internet, and enable end users to access and interact with it through a graphical user interface (GUI) or assistive technology.</p>
+      <p>This category does not include command-line tools and web clients intended to be used for software development and testing.</p>
```

Comment:
```suggestion
      <p>This category does not include command-line tools and web clients intended to be used primarily for software development and testing.</p>
```
(I think we should make sure we don't create a loophole for browsers that include excellent development functionality.)